Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -743,9 +743,11 @@ inline void BuilderCustom::applyInitialFontFamily(BuilderState& builderState)
         if (CSSValueID sizeIdentifier = fontDescription.keywordSizeAsIdentifier())
             builderState.setFontSize(fontDescription, Style::fontSizeForKeyword(sizeIdentifier, false, builderState.document()));
     }
+
+    auto value = AtomString("initial"_s);
+    initialDesc.setOneFamily(value);
     if (!initialDesc.firstFamily().isEmpty())
         fontDescription.setFamilies(initialDesc.families());
```

Comment:
```suggestion
    fontDescription.setFamilies(initialDesc.families());
```

This is what I originally suggested. Doesn't this work?

Reply:
I doubt your fix is correct, since it will instead look up a font named "initial".

Reply:
@nt1m unfortunately, It crash when remove the condition, It looks that we should add the font family manually, this is what I was trying in the modification, It worked for the case that I tested.

Reply:
I'm curious what crash you get, because the end result is actually the same, with the only difference that your version sets a bogus font-family before setting the correct one.

Reply:
When removed the condition like you sugested, It apply the property correctly but  when you pass the mouse over the element for exemple the crash happens

Reply:
From what I debugged the error happens because when created fontcascadedescription the font family is empty

Reply:
Do you have a crash trace?

Reply:
Only thing that appears for me is 
webcore: crashed in terminal

Reply:
should I compile the debug version, for get it?

Reply:
```
void setOneFamily(const AtomString& family) { ASSERT(m_families->size() == 1); m_families.get()[0] = family; }
```
m_families items most probably is the problem